test/e2e/cronjob.go

```diff
@@ -271,8 +367,17 @@ func newTestCronJob(name, schedule string, concurrencyPolicy batch.ConcurrencyPo
 			},
 		},
 	}
-	if sleep {
-		sj.Spec.JobTemplate.Spec.Template.Spec.Containers[0].Command = []string{"sleep", "300"}
+	if successfulJobsHistoryLimit != nil {
```

Comment:
These are both pointers, so you can assign it always, they should get defaulted properly by the api server.